### ISSUE TITLE
BICUBIC interpolation used for linear probing

### ISSUE THREAD

**Issue by honggudrnjs** [rdangovs/essl#2]
Thx for sharing amazing work and codesets in advance while I have a single question on the code.
In 'eval_loop' function of 'main.py', bicubic interpolations are used for data preprocessing on both train and test sets.
Is there any background for this? Why bicubic interpolations are used (even upscaling for train data processing)?

**Comment by rdangovs** [comment]
Hi, thanks for reaching out. In my understanding, BICUBIC interpolation is useful when enlarging the images. The train set uses random resized cropping, which selects crops, smaller than the original size of the image, so the crop is enlarged via interpolation to 32x32. Likewise, the test set is enlarged from 32x32 to 36x36 before center cropping (works slightly better than using the original image). All of the above are not firm choices, one could play with the details for better performance. Further experiments about similar details should be available here: https://github.com/rwightman/pytorch-image-models.

Happy to chat more!
